integration/tests/api/event_test.go

```diff
@@ -52,24 +51,25 @@ func TestStreamingEventInformationIsReturned(t *testing.T) {
 		},
 	})
 
+	t.Logf("containerId in event_test is %v", containerId)
 	timeout := make(chan bool, 1)
 	go func() {
 		time.Sleep(60 * time.Second)
 		timeout <- true
 	}()
 
-	select {
-	case ev := <-einfo:
-		if ev.EventType == 0 {
-			marshaledData, err := json.Marshal(ev.EventData)
-			require.Nil(t, err)
-			var oomEvent *oomparser.OomInstance
-			err = json.Unmarshal(marshaledData, &oomEvent)
-			require.Nil(t, err)
-			require.True(t, oomEvent.ProcessName == "stress")
+	for {
+		select {
+		case ev := <-einfo:
+			if ev.EventType == 0 {
+				if strings.Trim(ev.ContainerName, "/ ") == strings.Trim("docker/"+containerId, "/ ") {
```

Comment:
I am running into issues using the Client().ContainerInfo() and DockerContainer() methods with this image type because I keep getting the error "failed to get container..." with error "unknown container...." I assume this is because I force the container to timeout and generally the client's attempt to get the container information occurs after the container has stopped. However, the container id is returned when the "-d" flag is added to the docker command, so would it be sufficient to check that the event container name contains the returned containerId?